Framework/Algorithms/test/DetectorEfficiencyCorTest.h

```diff
@@ -117,14 +119,9 @@ class DetectorEfficiencyCorTest : public CxxTest::TestSuite {
     space->getAxis(0)->unit() = UnitFactory::Instance().create("DeltaE");
     Workspace2D_sptr space2D = boost::dynamic_pointer_cast<Workspace2D>(space);
 
-    BinEdges x{0.0, 0.0, 0.0, 0.0, 4.0};
-    // Fill a couple of zeros just as a check that it doesn't get changed
-    std::vector<double> y{10, 11, 12, 0};
-    std::vector<double> e{sqrt(5.0), sqrt(5.0), sqrt(5.0), 0.0};
-
-    space2D->setBinEdges(0, x);
-    space2D->mutableY(0).assign(y.begin(), y.end());
-    space2D->mutableE(0).assign(e.begin(), e.end());
+    space2D->setHistogram(0, BinEdges{0.0, 0.0, 0.0, 0.0, 4.0},
+                          Counts{10, 11, 12, 0},
+                          CountVariances{sqrt(5.0), sqrt(5.0), sqrt(5.0), 0.0});
```

Comment:
If you use `CountVariances` you have to remove the `sqrt`. `E` is standard deviations be default.